packages/ra-core/src/core/Resource.tsx

```diff
@@ -14,19 +15,46 @@ export const Resource = (props: ResourceProps) => {
         <ResourceContextProvider value={name}>
             <Routes>
                 {create && (
-                    <Route path="create/*" element={getElement(create)} />
+                    <Route
+                        path="create/*"
+                        element={
+                            <CanAccess action="create" resource={name}>
```

Comment:
*Note*: the *authentication* control is done by the page controllers. the *authorization* control is done by the parent resource. This may be unexpected. 

I know that we chose to do that to avoid enabling / disabling queries in the controller depending on the authorization, but developers will be surprised that using `useCreateController` alone won't check for the authorization. 

We need to discuss or document this one.